src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/calculator';
 import './App.css';
 
 function App() {
```

Comment:
- [ ] Your main component **is not class-based**. Kindly ensure that _App.js_ is class-based.